### PR TITLE
crepes that can have food items folded in them

### PR DESCRIPTION
**Pull request by klushy225**
[Catering]
[Feature]
[Game object]

## About the PR 
Adds crepes that can have other food items folded into them. Crepe batter is made by combining two milk bottles, one egg, and one sweet dough in the mixer. The batter is then cooked on low for 5 seconds in the oven. Other ingredients can be added in the oven with the batter or folded into the crepe after it is cooked by clicking on it with the ingredient.

SPRITES: https://imgur.com/a/0io2HRL

it marked my crepes as erotic lmfao

## Why's this needed? 

Someone ordered crepes while I was chef the other day, and there were no crepes. and I like crepes.

## Changelog 

```changelog
(u)Klushy225
(*)Added crepes to the game, which can have other ingredients folded into them
```

### CODE REVIEW

**Comment by deathrobotpunch** [comment]
can I put a crepe in a crepe and create crepeseption?

**Comment by klushy225** [comment]
> 

of course!

File: code/datums/cookingrecipes.dm [hunk 1]
```diff
@@ -1656,11 +1666,46 @@ datum/cookingrecipe/flockbrainburger
...
+			return null
+		var/obj/item/reagent_containers/food/snacks/custom_crepe = new
+		custom_crepe_food.reagents.trans_to(custom_crepe, 50)
+		if(custom_crepe.real_name)
+			custom_crepe.name = "[custom_crepe_food.real_name] folded in a crepe"
+		else
+			custom_crepe.name = "[custom_crepe_food.name] folded in a crepe"
+
+		var/icon/I = new/icon('icons/obj/foodNdrink/food_dessert.dmi', "folded-crepe")
+		//I.Blend(custom_crepe_food.get_food_color(), ICON_ADD)
```
Comment (ZeWaka) [inline code/datums/cookingrecipes.dm:1703]: commented out code?

Comment (klushy225) [reply code/datums/cookingrecipes.dm:1703]: forgot about that, fixing it now

**Comment by deathrobotpunch** [comment]
best PR yet

**Comment by wolfolotl** [comment]
1. can we see a screenshot of how they look
2. this might be out of scope but what are your thoughts on using an icing bag on a crepe to let you drizzle it with whatever's inside

**Comment by klushy225** [comment]
> 1. can we see a screenshot of how they look
> 2. this might be out of scope but what are your thoughts on using an icing bag on a crepe to let you drizzle it with whatever's inside

1. sure I'll post the sprites now

2. I can try, but this is only my second PR so we'll see

**Comment by deathrobotpunch** [comment]
the crepes are too erotic for my eyes you should have added an NSFW warning to protect the younger ones

**Comment by wolfolotl** [comment]
could probably make the finished crepes a bit bigger, item sprite scale's pretty loose but they seem to be about the size of tortilla chips currently

File: code/datums/cookingrecipes.dm [hunk 2]
```diff
@@ -1656,11 +1666,46 @@ datum/cookingrecipe/flockbrainburger
...
+
+		var/obj/item/reagent_containers/food/snacks/custom_crepe_food
+		for (var/obj/item/reagent_containers/food/S in ourCooker.contents)
+			if (S.type == item1)
+				continue
+			custom_crepe_food = S
+			break
+		if(!custom_crepe_food)
+			return null
+		var/obj/item/reagent_containers/food/snacks/custom_crepe = new
```
Comment (TobleroneSwordfish) [inline code/datums/cookingrecipes.dm:1695]: this should be a crepe and not an abstract snack object

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
Sorry for the late review, I think this one slipped through the cracks!

**Comment by klushy225** [comment]
sorry for kinda forgetting about this- I took a break from SS13 and life got in the way. I'm working on fixing it up now

**Comment by jan-antilles** [comment]
I'm not really a fan of the sprites, they look kinda mushy and indistinct. And personally I would rather have crepes be a nice dish made with Botany fruit, instead of "heehoo food crimes," but that's mostly just me.

**Comment by klushy225** [comment]
> I'm not really a fan of the sprites, they look kinda mushy and indistinct. And personally I would rather have crepes be a nice dish made with Botany fruit, instead of "heehoo food crimes," but that's mostly just me.

I could add some special recipes with cream and fruits on top and such. The sprite work is very temporary

**Pull request closed without merging**